flit/wheel.py

```diff
@@ -155,6 +156,9 @@ def write_metadata(self):
         elif self.ini_info['entry_points_file'] is not None:
             self._add_file(self.ini_info['entry_points_file'],
                            dist_info + '/entry_points.txt')
+        for name in ('LICENSE', 'COPYING'):
+            for fname in glob.glob(name + '*'):
```

Comment:
Can we sort the globbed files? For reproducible builds, we want to ensure files are added in a predictable order, and I don't think there's any guarantee about the return order from glob().

Also, this currently takes files from the working directory, which is usually but not always correct. `self.directory` is a Path object, so you should be able to use its [glob method](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob).